jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/DatagramChannelEndPoint.java

```diff
@@ -46,6 +46,21 @@ public DatagramChannel getChannel()
         return (DatagramChannel)super.getChannel();
     }
 
+    @Override
+    public SocketAddress getRemoteSocketAddress()
+    {
+        try
+        {
+            return getChannel().getRemoteAddress();
+        }
+        catch (Exception e)
+        {
+            if (LOG.isTraceEnabled())
+                LOG.trace("ignored", e);
+        }
+        return null;
```

Comment:
Can we return HttpConfiguration.remoteAddress here? (not sure, as this is deep in jetty-io)

Reply:
No need to.  We are applying HttpConfiguration.localAddress at the level of the Connection, not the endpoint.   There is not HttpConfiguration.remoteAddress